Backend/Remora.Discord.API.Abstractions/API/Rest/IDiscordRestApplicationAPI.cs

```diff
@@ -256,9 +264,10 @@ Task<Result<IApplicationCommand>> EditGuildApplicationCommandAsync
         Optional<string> name = default,
         Optional<string> description = default,
         Optional<IReadOnlyList<IApplicationCommandOption>?> options = default,
-        Optional<bool> defaultPermission = default,
         Optional<IReadOnlyDictionary<string, string>?> nameLocalizations = default,
         Optional<IReadOnlyDictionary<string, string>?> descriptionLocalizations = default,
+        Optional<IDiscordPermissionSet> defaultMemberPermissions = default,
+        Optional<bool?> dmPermission = default,
```

Comment:
dm_permission is not available for guild commands (see discord/discord-api-docs#4861). Remove this parameter.

Reply:
Oops, right